src/app/(proper_react)/redesign/(authenticated)/user/dashboard/subscribed/page.tsx

```diff
@@ -48,13 +48,14 @@ export default async function Subscribed() {
   const scans = await getAllScanResults(profileId);
 
   // In dev mode, record scans every time this page is reloaded.
-  // The webhoook does this in production.
```

Comment:
I, for one, will be sad for this typo to disappear.
I like to imagine it's more of a battle cry, webhoooooook!

Reply:
![](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExbWZ3bXhvcjVrb2p4YXRkNWFoeHNlNDk0MHlrejN1Z2d5OWV5dTcyMiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/9p8hgZoQjj9y8/giphy.gif)